### PR TITLE
Extracted not added items from PR #313

### PR DESCRIPTION
**Pull request by Ihromant**
https://github.com/konsoletyper/teavm/pull/313/files
lots of items already were implemented, so I extracted everything that was not implemented. Did not merge it, but I suppose that is should be authored by @niloc132

### CODE REVIEW

File: jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java [hunk 1]
```diff
@@ -30,7 +30,17 @@ public interface DOMTokenList extends JSObject {
 
     void add(String token);
 
+    void add(String token1, String token2);
+
+    void add(String... tokens);
```
Comment (konsoletyper) [inline jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java:35]: Are you sure that varargs supported in JSO?

Comment (Ihromant) [reply jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java:35]: There also was a comment to create issue to support varargs in JS. Ok, will create issue to support varagrs.

Comment (konsoletyper) [reply jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java:35]: So are they supported or not? If they aren't, I can't accept PR with these declarations

Comment (Ihromant) [reply jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java]: I thought that you know as author of TeaVM. But ok, will check.

Comment (konsoletyper) [reply jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java]: I'm not a machine remebering each and every character from this repository

Comment (Ihromant) [reply jso/apis/src/main/java/org/teavm/jso/dom/types/DOMTokenList.java]: Sorry, didn't want to offend you. Just checked. Varargs are not supported. So, extracted varargs to separate PR https://github.com/konsoletyper/teavm/pull/867

**Pull request closed without merging**